src/_grid.scss

```diff
@@ -0,0 +1,67 @@
+/* @TODO move to variables/settings file */
+$divisions: 'one' 1, 'two' 2 , 'three' 3, 'four' 4, 'five' 5, 'six' 6, 'seven' 7, 'eight' 8, 'nine' 9, 'ten' 10, 'eleven' 11, 'twelve' 12;
+
+/* 12 column grid */
+.grid {
+    display: grid;
+    grid-template-columns: repeat(12, [col] 1fr);
+    grid-auto-rows: minmax(50px, auto);
+    grid-gap: 1em;
+}
+
+/* Dynamic grid */
+.autofit {
```

Comment:
I think i preferred `autogrid` more, its more clear. What do you guys think @piccolazucca @kingsleyzissou?

Reply:
@dimitraz I was thinking of leaving the standard grid in for now, because the autogrid and the standard grid would have different applications.

The autogrid, for example, would be useless to have a sidebar column.